### ISSUE TITLE
[FEAT] :sparkles: Create UI for `login` page

### ISSUE THREAD

**Issue by aayushchugh** [MultiEmail/MultiEmail-admin#14]
### Description

![image](https://user-images.githubusercontent.com/69336518/194366779-f5e74366-b29c-435c-b40c-c87bd2419a0e.png)

**Comment by Chaitanya-087** [comment]
i would like to work on this

**Comment by abel-otegbola** [comment]
Hi @aayushchugh 
Can I work on this.

**Comment by aayushchugh** [comment]
I am currently creating folder structure, you can work after I create folder structure

**Comment by Tchakoumi** [comment]
I can do this

**Comment by KanLSK** [comment]
Wait ive already done this

**Comment by aayushchugh** [comment]
@KanLSK you did it for client It's admin

**Comment by KanLSK** [comment]
> @KanLSK you did it for client It's admin

It literally the same design

**Comment by aayushchugh** [comment]
@KanLSK Yup we can copy code from frontend repo

**Comment by Tchakoumi** [comment]
I think if it's same design, then the component be reused in the admin app
or what is it about?

**Comment by aayushchugh** [comment]
@Tchakoumi You can copy paste the code.

but let me finish the folder structure first

**Comment by Tchakoumi** [comment]
ok.

**Comment by aayushchugh** [comment]
@Tchakoumi after #15 is merged you can start working on the Login page

you can copy the code from frontend repo

**Comment by aayushchugh** [comment]
completed in #20
